packages/@lwc/engine-core/src/framework/rendering.ts

```diff
@@ -655,23 +693,13 @@ function createViewModelHook(elm: HTMLElement, vnode: VCustomElement, renderer:
     return vm;
 }
 
-/**
- * Collects all slots into a SlotSet, traversing through VFragment Nodes
- */
 function collectSlots(vm: VM, children: VNodes, cmpSlotsMapping: { [key: string]: VNodes }) {
     for (let i = 0, len = children.length; i < len; i += 1) {
         const vnode = children[i];
         if (isNull(vnode)) {
             continue;
         }
 
-        // Dive further iff the content is wrapped in a VFragment
-        if (isVFragment(vnode)) {
```

Comment:
This is no longer needed because we've flattened and removed all the VFragment nodes by the time we hit `allocateInSlot`.